run_bot.sh

```
@@ -646,7 +679,7 @@ test_api_request() {
     # Send API stop request
     print_info "🛑 Sending API stop request..."
     local api_response
-    api_response=$(docker exec "$(docker ps -q --filter ancestor=meet-teams-bot)" curl -s -X POST http://localhost:8080/stop_record \
+    api_response=$(docker exec "$(docker ps -q --filter ancestor=$(get_docker_image))" curl -s -X POST http://localhost:8080/stop_record \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)